source/reference/operator/aggregation/geoNear.txt

```diff
@@ -211,17 +211,19 @@ When using :pipeline:`$geoNear`, consider that:
 Examples
 --------
 
+.. include:: /includes/geospatial-places-data-setup.rst
+
 Maximum Distance
 ~~~~~~~~~~~~~~~~
 
 .. note::
 
    .. include:: /includes/extracts/4.2-changes-geoNear-limit.rst
 
-Consider a collection ``places`` that has a ``2dsphere`` index. The
-following aggregation uses :pipeline:`$geoNear` to find documents with
-a location at most 2 meters from the center ``[ -73.99279 , 40.719296
-]`` and ``category`` equal to ``Parks``.
+Consider the ``places`` collection above, it has a ``2dsphere`` index. 
```

Comment:
This is a [comma splice](https://www.grammarly.com/blog/comma-splice/) and should be rewritten.
```suggestion
The ``places`` collection above has a ``2dsphere`` index.
```